docs/docs/index.md

```diff
@@ -1,143 +1,134 @@
 # `rsconnect-jupyter` User Guide
 
-`rsconnect-jupyter` is a plugin for Jupyter Notebooks that enables publishing notebooks to Posit Connect.
+The `rsconnect-jupyter` package is a _Jupyter Notebook_ extension (i.e., `nbextension`) that provides publishing compatiblity with [Posit Connect](https://docs.posit.co/rsc/#rstudio-connect).
 
 ## Requirements
 
-- Python 3.5.0 and higher
-- Jupyter Notebook 5.x
-- [pip](https://pypi.org/project/pip/)
-- [wheel](https://pypi.org/project/wheel/)
-- [Posit Connect](https://www.posit.co/download/posit-connect/) v1.7.0 or higher, configured with Python support
+- [Python >=3.7](https://www.python.org/downloads/)
+- [Jupyter](https://pypi.org/project/jupyter)
+- [Posit Connect](https://www.posit.co/download/posit-connect/**) v1.7.0 or higher, configured with Python support
```

Comment:
@mmarchetti - should I update the Connect version requirement? I don't think v1.7.0 is still available.

Reply:
Yes, let's remove the version requirement since all supported Connect versions can run Python (if it's configured).

Reply:
Our oldest supported version is the 2021.08 release, so it's probably fine to bump to that at least: https://posit.co/support/#supported-connect-versions